### DISCUSSION TITLE
Highlighting typed text

### DISCUSSION THREAD

**Discussion by LordMcFly** [scambier/obsidian-omnisearch#273] [Ideas]
Heyo! First, you have my infinite gratitude for this super useful plugin with perfect UI (yes, I use the word perfect because it is both simple and fully detailed). The only drawback when compared to the default search is that when I open the file, I find myself in front of a big wall of text with no idea where the text I searched actually is.

So maybe you could add a simple highlight when the file is opened, either that lasts a few seconds or is cancelled when you press escape. That would be awesome.

Thanks again and keep the good work!

**Comment by scambier** [comment]
Hello and thank you :)

This is already a waiting feature (#65). I'll to re-check the API and find examples in other plugins
